.buildkite/commands/package_windows.ps1

```diff
@@ -0,0 +1,50 @@
+# Stop script execution when a non-terminating error occurs
+$ErrorActionPreference = "Stop"
+
+& "prepare_windows_host_for_node.ps1"
```

Comment:
This comes from the CI Toolkit Buildkite plugin.

Reply:
Thanks for documenting that. Wonder if that would be worth a code comment?